systemtest/ingest_test.go

```diff
@@ -300,3 +307,60 @@ func TestIngestPipelineBackwardCompatibility(t *testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+func TestMobileLogsWithGeoLocation(t *testing.T) {
```

Comment:
nit: imo the test fits marginally better in otlp_test.go although this specific PR is related to ingest pipeline. This is just a special case of TestOTLPGRPCLogs. Maybe call it `TestOTLPGRPCLogsClientIP`.

Reply:
You mean to rename it and also move it to `otlp_test.go` or to just rename it?

Reply:
move + rename